tests/system/python/api/test_audit.py

```diff
@@ -110,5 +148,26 @@ def test_create_audit_entry(self, fledge_url, payload, total_count):
         assert 200 == r.status
         r = r.read().decode()
         jdoc = json.loads(r)
+
+        # Storage category when with SQLite plugin, creates a child category `sqlite` hence one additional CONAD entry.
+        # CONAD, INFORMATION, '{{"name":"sqlite", ...}'
+        # so total_count is being decresed in case of 'postgres' storage engine
```

Comment:
Interesting one! It's by design?

Reply:
Let @MarkRiddoch confirm on this. I think yes. When storage is configured with Postgres there are no children (child category) for Storage but SQLite only.

Reply:
Yes, only SQLite has a child category